index.html

```diff
@@ -2403,6 +2408,98 @@ <h3>
           can get their games and apps rated with IARC</a>.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>shortcuts</code> member
+        </h3>
+        <p>
+          The <dfn>shortcuts</dfn> member is an <a>array</a> of
+          <a>ShortcutItem</a>s that provide access to key tasks within a web
+          application. For example, they can be used to link directly to a
+          user's timeline within a social media application or their recent
+          orders in an e-commerce context.
+        </p>
+        <p>
+          A developer may provide as many shortcuts as they like, but the user
```

Comment:
Shouldn't use the keyword "may" when this is not a normative requirement.

I would just drop the first clause and rewrite this sentence. Suggestion: "How shortcuts are presented, and how many of them are shown to the user, is at the discretion of the user agent and/or operating system."